ValetTests/ValetTests.m

```diff
@@ -110,8 +110,11 @@ - (void)test_initialization_invalidArgumentsCauseFailure;
 {
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"" accessibility:VALAccessibilityAlways]);
     XCTAssertNil([[VALValet alloc] initWithIdentifier:@"test" accessibility:0]);
-    XCTAssertNil([[VALSynchronizableValet alloc] initWithIdentifier:@"test" accessibility:VALAccessibilityWhenPasscodeSetThisDeviceOnly]);
```

Comment:
Ok, cool. Just asking since `VALAccessibilityWhenPasscodeSetThisDeviceOnly` went away and my pattern match alarm went off.